client/tests/webdriver/baseSpecs/createNewReport.spec.js

```diff
@@ -105,6 +105,25 @@ describe("When creating a report", () => {
     await validateAttendee(ADVISOR, EXPECTED_DATA.now, "advisor")
     await validateAttendee(INTERLOCUTOR, EXPECTED_DATA.now, "interlocutor")
   })
+
+  it("Should switch an atendee between an advisor and interlocutor", async() => {
+    const advisor = await CreateReport.getPersonByName(ADVISOR)
+    // change atendee to interlocutor
+    expect(await advisor.advisorCheckbox.isExisting()).to.equal(true)
+    expect(await advisor.advisorCheckbox.isSelected()).to.equal(false)
+    advisor.advisorCheckbox.click()
+    await browser.pause(500)
```

Comment:
Please add a comment indicating that this pause is to wait for the attendee to be moved to the other section.